src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Machine> get() {
+        return List.of(new Excavator(), new Excavator());
```

Comment:
Consider creating a local variable for the list you're returning. It can make debugging easier and the code more readable.